### PR TITLE
FORMAT_AGREEMENTS

### PR DESCRIPTION
**Pull request by srocillo23**
Format we all are going to follow to keep our files organized.

### CODE REVIEW

**Review by markossmdz — APPROVED** [review]
I agree

**Review by TiagoLezcano — APPROVED** [review]
Nice, we will follow this format now on.

**Review by delgueero — APPROVED** [review]
I like it 👍

File: FORMAT_AGREEMENTS [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Brief introduction about the field we are talking about
+
+---
+
+**1. Header Guidelines**  
+
+Headers must clearly define the topic.
+
+# Main Title
```
Comment (bananas24) [inline FORMAT_AGREEMENTS:12]: This is bigger than "our file title" Change it

**Pull request closed without merging**